packages/theme_tailor/README.md

```diff
@@ -153,6 +153,8 @@ The generator will create a static getter with a list of the generated themes:
 final allThemes = MyTailorGeneratedTheme.themes;
 ```
 
+If `themes` property is already used in the tailor class, the generator will use another name and print a warning.
+
```

Comment:
It's already on the develop, why it shows as a change here? Can you fix the merge?